naughty/fedora-30/268-reportd-event-collect

```diff
@@ -0,0 +1 @@
+*reportd task for workflow workflow_FedoraCCpp did not finish: Event collect_GConf has no handlers defined*
```

Comment:
This would work only sometimes. This comes from wildcard expansion for `collect_*` and therefore we should do the same.
Random example where it would not be enough is for example [this one](https://logs-https-cockpit.apps.ci.centos.org/logs/pull-258-20191120-121616-df5419e0-cockpit-project-cockpit--fedora-31/log.html#49-2)

Reply:
Ack, thanks. Updated!